onlinebanking/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2022 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by atef on 9/8/2022.
+  -->
+
+<manifest package="com.adyen.checkout.onlinebanking">
```

Comment:
Can we use this package name? If `PL` is different than the other components then we probably need to change the package name as well